be1-go/channel/lao/mod.go

```diff
@@ -477,6 +504,15 @@ func (c *Channel) processRollCallObject(action string, msg message.Message) erro
 	return nil
 }
 
+func createChirpingChannel(publicKey string, channelID string, hub channel.HubFunctionalities,
+							general *generalChriping.Channel, log zerolog.Logger) {
```

Comment:
this formatting is not compliant with `gofmt`. Please ensure that your code is formatted with `gofmt`.

Reply:
How can I do this? Is there a cmd for this?

Reply:
ok now we can just do "make fmt" thanks to Celine